### PR TITLE
Feat/add information button

### PR DESCRIPTION
**Pull request by MaxPostema**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
-  Clean commits
- [x] No warnings during install
- [x] Updated flow typing

### CODE REVIEW

**Review by chinook25 — APPROVED** [review]

File: src/components/DescriptionUrls.vue [hunk 1]
```diff
@@ -12,16 +12,34 @@ export default {
...
   methods: {
+    // Returns an array of structure [{text, url}, {text, url}, ...]
+    // Note: split may be [{text}] if no url is found
+    textURLSplit (text) {
+      const split = text.split(/(?:[^\S]|^)((?:(?:https?:\/\/)|(?:www\.))(?:\S+))/gi)
+      return split.reduce((accumulator, current, index) => {
+        if (index % 2) {
+          return accumulator
+        } else {
+          return [...accumulator, { text: current, url: split[index + 1] }]
```
Comment (jelmerveen) [inline src/components/DescriptionUrls.vue:33]: what is in the accumulator

File: src/components/DescriptionUrls.vue [hunk 2]
```diff
@@ -12,16 +12,34 @@ export default {
...
+        }
+      }, [])
+    },
     ExternalURL (url) {
       if (!url.match(/^[a-zA-Z]+:\/\//)) {
-        url = 'https://' + url
+        return 'https://' + url
```
Comment (jelmerveen) [inline src/components/DescriptionUrls.vue:39]: what does this actually do? only check for protocol? then you could better check this regex: /^http/gmi

File: test/unit/specs/field-types/MultiSelectFieldComponent.spec.js [hunk 3]
```diff
@@ -101,28 +101,26 @@ describe('MultiSelectFieldComponent unit tests', () => {
...
+      { id: 'ref3', label: 'label3', value: 'ref3' }
+    ])
   })
 
   it('should fetch a value that is outside the first range on creation', async () => {
     propsData.value = ['ref1', 'ref2', 'ref10']
     const wrapper = mount(MultiSelectFieldComponent, {
       propsData: propsData,
-      stubs: { 'fieldMessages': '<div>This field is required</div>' }
+      stubs: { fieldMessages: '<div>This field is required</div>' }
```
Comment (jelmerveen) [inline test/unit/specs/field-types/MultiSelectFieldComponent.spec.js:123]: you could declare fieldMessages as a constant, and then only add the property like:

```const fieldMessages = '<div>This field is required</div>'```


```stubs: { fieldMessages }```

Saves a lot of copy paste

**Review by jelmerveen — CHANGES_REQUESTED** [review]

File: src/components/Description.vue [hunk 4]
```diff
@@ -29,38 +33,51 @@ export default {
...
+      long: info.text.substring(enterLocation + 1),
+      info: info.information
+    }
+  } else {
+    return { normal: info.text, long: '', info: info.information }
+  }
+}
+
+function useInternationalisation ($t) {
+  return $t &&
```
[the remark this answers is not in the document]
Comment (chinook25) [reply src/components/Description.vue:69]: If it exists and the values are not set, it should use default values, thus the function returns false. At least that is what I understand from this code :)

**Review by jelmerveen — APPROVED** [review]

**Pull request merged**